app/uk/gov/hmrc/gform/testonly/TestOnlyController.scala

```diff
@@ -54,6 +55,8 @@ import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 import uk.gov.hmrc.gform.views.html.hardcoded.pages.{ destinations, save_form_page, snapshot_page, snapshots_page, update_snapshot }
```

Comment:
I have no other place to put this.

`snapshots_page.scala.html`  takes unused `formId: String` param. Let's fix it.